sklearn/metrics/scorer.py

```diff
@@ -477,6 +477,9 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 neg_median_absolute_error_scorer = make_scorer(median_absolute_error,
                                                greater_is_better=False)
+neg_root_mean_sqr_error_scorer = make_scorer(mean_squared_error,
```

Comment:
let's use ``neg_root_mean_squared_error_scorer`` and ``neg_root_mean_squared_error``